libs/application/templates/new-primary-school/src/lib/dataSchema.ts

```diff
@@ -63,11 +63,15 @@ export const dataSchema = z.object({
   relatives: z
     .array(
       z.object({
-        fullName: z.string().min(1),
-        phoneNumber: phoneNumberSchema,
-        nationalId: z.string().refine((n) => kennitala.isValid(n), {
-          params: errorMessages.nationalId,
+        NationalIdWithName: z.object({
+          name: z.string().min(1),
+          nationalId: z
+            .string()
+            .refine((nationalId) => kennitala.isValid(nationalId), {
+              params: errorMessages.nationalId,
+            }),
```

Comment:
Maybe create a new const `nationalIdWithNameSchema` to remove duplicated code (Like `phoneNumberSchema`)? 🤔 

```suggestion
        NationalIdWithName: nationalIdWithNameSchema,
```